packages/python-minikerberos/PKGBUILD

```diff
@@ -26,5 +26,9 @@ package() {
   cd "$_pkgname-$pkgver"
 
   python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+  
+  # workaround until https://github.com/skelsec/minikerberos/pull/30 is merged
+  # and the python source package is updated
+  rm -rf "$pkgdir/usr/lib/python3.10/site-packages/tests"
```

Comment:
If you make a change you need to bump `pkgrel`

Reply:
bumped